explorer/src/components/account/AnchorAccountCard.tsx

```diff
@@ -63,10 +63,13 @@ export function AnchorAccountCard({ account, program }: { account: Account, prog
             </thead>
             <tbody className="list">
               {decodedAnchorAccountData &&
-                Object.keys(decodedAnchorAccountData).map((key) => {
-                  // @ts-ignore
-                  return renderAccountRow(key, decodedAnchorAccountData[key]);
-                })}
+                Object.keys(decodedAnchorAccountData).map((key) => (
+                  <AccountRow
+                    objectKey={key}
```

Comment:
React needs the keys to find themselves on the actual elements in this array, not on the inner `tr`. You don’t even need to make `AccountRow` accept a `key` prop! It’s a synthetic prop that’s always accepted, on any element.

This PR should be raising key errors at this location right now. Check out my code suggestions from yesterday where you add key here and only make `AccountRow` accept a `value` prop.

Reply:
I think `key` in this context is actually the variable name that is displayed in the interface. For clarity, it could probably be called `variableName`?

<img width="468" alt="Screen Shot 2022-04-02 at 12 36 11 PM" src="https://user-images.githubusercontent.com/374048/161392789-7afa6bfa-3b8d-449e-88e4-29ff9d83967b.png">

Reply:
@steveluscher good catch. Although the built website did not have issues with the card account name key, I addressed your change in latest commit.

Reply:
Yeah. I don’t think React raises key warnings in the prod build.

Reply:
Oh, of course you’re right, @coudron. That’s what I get for reviewing PRs from the can.